Examples/Framework/ML/src/NeuralCalibrator.cpp

```diff
@@ -107,6 +108,17 @@ void ActsExamples::NeuralCalibrator::calibrate(
         Acts::ActsSquareMatrix<Acts::eBoundSize> fcov =
             E * measurement.covariance() * E.transpose();
 
+        Acts::Vector3 dir = Acts::makeDirectionFromPhiEta(
+            fpar[Acts::eBoundPhi], fpar[Acts::eBoundTheta]);
+        Acts::Vector3 gpos =
+            surf.localToGlobal(gctx, fpar.segment<2>(Acts::eBoundLoc0), dir);
+        Acts::RotationMatrix3 rot =
```

Comment:
```suggestion
// Rotation matrix. When applied to global coordinates, they are rotated into the local reference frame of the surface. Note that this such a rotation can be found by inverting a matrix whose columns correspond to the coordinate axes of the local coordinate system.
        Acts::RotationMatrix3 rot =
```

Better double check if I got the explanation right

Reply:
Sounds about right